requirements/base.in

```diff
@@ -22,6 +22,7 @@ djangorestframework-camel-case
 drf-yasg
 drf-writable-nested
 vng-api-common
+gemma-zds-client<1.0
```

Comment:
Waarom is de GEMMA ZDS client nodig voor de standaard?

Reply:
die wordt gebruikt om bijvoorbeeld het zaaktype op te halen bij het aanmaken van een zaak

Reply:
Hebben we het dan over https://github.com/VNG-Realisatie/gemma-zds-client ?

Reply:
Dat klopt, het gaat om die repository.

Er is een afhankelijkheid in de zin dat de referentie-implementatie niet werkt zonder die library